playbooks/portals-setup-initial.yml

```diff
@@ -38,25 +38,22 @@
   hosts: webportals
   remote_user: "root"
   gather_facts: False
-  serial: 1              # Limit concurrency
+  serial: 1 # Limit concurrency
   any_errors_fatal: True # Stop on first error, do not execute on the next host
   vars:
     lastpass_required: True
   tasks:
     - name: Skip if initial root like user is defined
       meta: end_host
       when: initial_root_like_user is defined
-    
+
     # Check host OS version
     - name: Gather ansible facts on demand
       ansible.builtin.setup:
-    
-    - name: Check supported OS and version
-      assert:
-        that:
-          - ansible_distribution|lower == 'debian' and ansible_distribution_version == '10'
-        fail_msg: "{{ ansible_distribution }} {{ ansible_distribution_version }} is not yet supported by this role"
-    
+
+    - name: Check server OS compatibility
```

Comment:
same

Reply:
renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c